### PR TITLE
CSOAR-1149: Add csoar-automation-bridge on Docker documentation

### PR DESCRIPTION
**Pull request by ffelici-sumo**
Signed-off-by: Fabio Felici <ffelici@sumologic.com>

## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request introduce the documentation for CSOAR Automation Bridge on Docker for teach how to run a dockerized version of cloud soar automation bridge.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/cloud-soar/cloud-soar-bridge.md [hunk 1]
```diff
@@ -224,3 +224,70 @@ WantedBy=multi-user.target
...
+When run, the automation bridge listens on the Docker Unix socket for be able to execute the soar integration or can run a standalone daemon.
+
+Csoar Automation Bridge need to be able to communicate with the Docker API to works.
+
+### Prerequisites and configuration
+
+|Environment Variable                |Description    |Default   |
+|------------------------------------|---------------|----------|
+|`API_URL_HERE`                    |To determine which is the correct SOAR_URL, see [Sumo Logic Endpoints by Deployment and Firewall Security](https://help.sumologic.com/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security) and get the URL under the API Endpoint column. For example: https://api.eu.sumologic.com/api/||
+|`SOAR_TOKEN_HERE`                   |Log in to Sumo Logic and create a new [installation token](https://help.sumologic.com/docs/manage/security/installation-tokens/) with the name prefix csoar-bridge-token.||
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-bridge.md:245]:
```suggestion
|`API_URL_HERE`                    |To determine which is the correct SOAR_URL, see [Sumo Logic Endpoints by Deployment and Firewall Security](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security) and get the URL under the API Endpoint column. For example: https://api.eu.sumologic.com/api/||
|`SOAR_TOKEN_HERE`                   |Log in to Sumo Logic and create a new [installation token](/docs/manage/security/installation-tokens/) with the name prefix csoar-bridge-token.||
```

File: docs/cloud-soar/cloud-soar-bridge.md [hunk 2]
```diff
@@ -224,3 +224,70 @@ WantedBy=multi-user.target
...
+docker run -d \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-e SOAR_URL=API_URL_HERE \
+-e SOAR_TOKEN=SOAR_TOKEN_HERE \
+-e SIEM_URL=SIEM_URL_HERE \
+-e ALIAS=BRIDGE_ALIAS_HERE \
+-e DOCKER_TLS_CERTDIR=/certs \
+-v docker-certs-ca:/certs/ca -v docker-certs-client:/certs/client \
+public.ecr.aws/u5z5f8z6/sumologic/csoar-automation-bridge:latest
+ ```
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-bridge.md:265]:
```suggestion
```bash
docker run -d \
-v /var/run/docker.sock:/var/run/docker.sock \
-e SOAR_URL=API_URL_HERE \
-e SOAR_TOKEN=SOAR_TOKEN_HERE \
-e SIEM_URL=SIEM_URL_HERE \
-e ALIAS=BRIDGE_ALIAS_HERE \
-e DOCKER_TLS_CERTDIR=/certs \
-v docker-certs-ca:/certs/ca -v docker-certs-client:/certs/client \
public.ecr.aws/u5z5f8z6/sumologic/csoar-automation-bridge:latest
```
```

File: docs/cloud-soar/cloud-soar-bridge.md [hunk 3]
```diff
@@ -224,3 +224,70 @@ WantedBy=multi-user.target
...
+|`SIEM_URL_HERE`           |The HTTP Sumo Logic collector to send the bridge logs|NONE|
+|`BRIDGE_ALIAS_HERE`               |Provide the alias name. With bridge ALIAS, it is possible to distinguish which integration resources will be executed with this automation bridge. When a new integration resource is created or edited, it is possible to select the default ALIAS or to create a new one. So every automatic action configured to use this resource will be performed with the Bridge that has the same ALIAS.|NONE|
+
+### Can run with two methodologies
+
+#### With the docker socket mounted as volume
+
+[Recommended]
+
+ ```bash
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-bridge.md:255]:
```suggestion
```bash
```

File: docs/cloud-soar/cloud-soar-bridge.md [hunk 4]
```diff
@@ -224,3 +224,70 @@ WantedBy=multi-user.target
...
+docker run -d \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-e SOAR_URL=API_URL_HERE \
+-e SOAR_TOKEN=SOAR_TOKEN_HERE \
+-e SIEM_URL=SIEM_URL_HERE \
+-e ALIAS=BRIDGE_ALIAS_HERE \
+-e DOCKER_TLS_CERTDIR=/certs \
+-v docker-certs-ca:/certs/ca -v docker-certs-client:/certs/client \
+public.ecr.aws/u5z5f8z6/sumologic/csoar-automation-bridge:latest
+ ```
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-bridge.md:265]:
```suggestion
```
```

File: docs/cloud-soar/cloud-soar-bridge.md [hunk 5]
```diff
@@ -224,3 +224,69 @@ WantedBy=multi-user.target
...
+```bash
+docker run -d \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-e SOAR_URL=API_URL_HERE \
+-e SOAR_TOKEN=SOAR_TOKEN_HERE \
+-e SIEM_URL=SIEM_URL_HERE \
+-e ALIAS=BRIDGE_ALIAS_HERE \
+-e DOCKER_TLS_CERTDIR=/certs \
+-v docker-certs-ca:/certs/ca -v docker-certs-client:/certs/client \
+public.ecr.aws/u5z5f8z6/sumologic/csoar-automation-bridge:latest
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-bridge.md:264]:
```suggestion
public.ecr.aws/u5z5f8z6/sumologic/csoar-automation-bridge:latest
```
```

File: docs/platform-services/automation-service/automation-service-bridge.md [hunk 6]
```diff
@@ -225,3 +229,70 @@ WantedBy=multi-user.target
...
+The image tagged `latest` and specific version to runs the automation bridge with the Sumo Logic Cloud SOAR Docker automation bridge executable.
+
+When run, the automation bridge listens on the Docker Unix socket for be able to execute the soar integration or can run a standalone daemon.
+
+Cloud SOAR Automation Bridge need to be able to communicate with the Docker API to works.
+
+### Prerequisites and configuration
+
+|Environment Variable                |Description    |Default   |
+|------------------------------------|---------------|----------|
```
Comment (kimsauce) [inline docs/platform-services/automation-service/automation-service-bridge.md:248]:
```suggestion
|:------------------------------------|:---------------|:----------|
```

**Review by kimsauce — COMMENTED** [review]
I've applied some formatting/style edits. Will let @jpipkin1 do the final signoff.

**Review by jpipkin1 — APPROVED** [review]
@ffelici-sumo, I made updates (see commit "[Revisions per John Pipkin review](https://github.com/SumoLogic/sumologic-documentation/pull/3729/commits/be3e2000c41ac7ca46b3fc46dd299021167c67f7)"). Let me know if these changes are okay.

**Comment by ffelici-sumo** [comment]
Changes are perfect, thanks!

**Pull request merged**